docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
+  <https://docs.plasmapy.org/en/latest/>`_.
+
+A preview of the documentation is generated every time a pull request
+is created or updated. You can access this preview by scrolling down
+to the checks at the bottom of a pull request, and clicking on
+``Details`` next to ``docs/readthedocs.org:plasmapy``.
 
 Building documentation
 ======================
-Documentation is built from the main branch on every commit pushed
-to it.
 
-Sphinx, the documentation generator of PlasmaPy, uses reStructuredText (reST) as its markup language. A primer on reST is available at this `webpage
+To install all dependencies required to develop PlasmaPy on your local
+computer, enter the top-level directory of the cloned repository and
+run
+
+.. code-block:: bash
+
+   pip install tox -r requirements.txt
+
+You can use `tox`_ to build the documentation from within the main
+PlasmaPy repository directory by running
+
+.. code-block:: bash
+
+   tox -e build_docs
+
+You can access the documentation landing page by opening
+``docs/_build/html/index.html`` with your browser of choice.
+
+When writing documentation, please make sure to fix any warnings that
+arise. To enforce this, the ``build_docs`` environment is set to fail
+on encountering any warnings via the ``-W`` flag to ``sphinx-build``.
+
+You can shorten the documentation build by running
+
+.. code-block:: bash
+
+   tox -e build_docs_no_examples
+
+in order to build the documentation without executing the
+:ref:`example notebooks <example_notebooks>`. This command will pass
+even if there are warnings.  The `tox`_ command may also include the
+``--parallel`` flag to allow a parallel build.
+
+If you have `make <https://www.gnu.org/software/make/>`_ installed,
+then you can build the documentation by entering the ``docs/`` directory
+and running
+
+.. code-block:: bash
+
+   make html
+
+Including the ``-j n`` flag in the ``make`` command will enable a
+parallel build, where ``n`` is replaced with the number of processes.
+
+Documentation tools
+===================
+
+ReStructuredText
+----------------
+
+PlasmaPy's documentation is written using the `reStructuredText (reST)
 <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
-of Sphinx's website.
+markup language. reST is human readable when viewed within a
+source code file or when printed out using `help`. reST also contains
+markup that allows the text to be transformed into `PlasmaPy's online
+documentation`_. reST files end in ``.rst``. Documentation contained
+within ``.py`` files is written in reST.
 
-Using sphinx within the project
--------------------------------
-To build docs locally, either:
+ReStructuredText Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* use `Tox <https://tox.readthedocs.io/en/latest/>`_ with ``tox -e build_docs`` from within the main PlasmaPy repository directory, or
-* enter the ``docs`` directory and run ``make html``.
+Here we show some examples of reST that are commonly used in PlasmaPy.
```

Comment:
Add a comment to the effect of "..., but please refer to the reST and sphinx documentation for a more complete list of available directives and syntax."